internal/discover/graphics.go

```diff
@@ -251,6 +312,26 @@ func optionalXorgDiscoverer(logger logger.Interface, driver *root.Driver, nvidia
 	return xorg
 }
 
+func buildXOrgSearchPaths(libRoot string) []string {
```

Comment:
Now that I see this as the nested for loops, it seems more difficult to actually understand what's being searched.

Does it make sense to flatten this instead and just do something like:
```
	return []string{
		filepath.Join(libRoot, "nvidia/xorg"),
		filepath.Join(libRoot, "xorg", "modules", "drivers"),
		filepath.Join(libRoot, "xorg", "modules", "extensions"),
		filepath.Join(libRoot, "xorg", "modules/updates", "drivers"),
		filepath.Join(libRoot, "xorg", "modules/updates", "extensions"),
		filepath.Join("/usr/lib/xorg", "modules", "drivers"),
		filepath.Join("/usr/lib/xorg", "modules", "extensions"),
		filepath.Join("/usr/lib/xorg", "modules/updates", "drivers"),
		filepath.Join("/usr/lib/xorg", "modules/updates", "extensions"),
		...         
	}
```

Reply:
I guess so.